README.md

```diff
@@ -5,16 +5,38 @@ You write it, and we deliver it.
 [![Coverage Status](https://coveralls.io/repos/ciudadanointeligente/write-it/badge.png?branch=master)](https://coveralls.io/r/ciudadanointeligente/write-it)
 [![Code Health](https://landscape.io/github/ciudadanointeligente/write-it/master/landscape.png)](https://landscape.io/github/ciudadanointeligente/write-it/master)
 
-Write-it is an application that aims to deliver messages to people whose contacts are to be private or the messages should be public, for example: members of congress. 
+Write-it is an application that aims to deliver messages to people whose contacts are to be private or the messages should be public, for example: members of congress.
 
 Write-it is a layer on top of [popit](http://popit.mysociety.org) from where it takes the people and adds contacts. The way it delivers messages is using plugins for example: mailit. And this approach allows for future ways of delivering for example: twitter, whatsapp, fax or pager.
 
 Future uses are in [congresoabierto](http://www.congresoabierto.cl) to replace the old "preguntales" (You can [check here](http://congresoabierto.cl/comunicaciones), to see how it used to work) feature, it was inspired by [writetothem](http://www.writetothem.com/).
 
 
+Quick Installation (Vagrant)
+============================
 
-Installation
-------------
+Assuming [you have Vagrant installed](http://docs.vagrantup.com/v2/installation/), run the following:
+
+    git clone git@github.com:ciudadanointeligente/write-it.git
```

Comment:
Probably best to use the http url here since it's a public repo, save people having to worry about ssh keys etc

```
git clone https://github.com/ciudadanointeligente/write-it.git
```